src/IssueViz.Security/Taint/TaintList/TaintVulnerabilitiesControl.xaml

```diff
@@ -1,15 +1,124 @@
+                  SelectedItem="{Binding SelectedIssue, Mode=TwoWay}"
+                  ItemsSource="{Binding Issues}"
+                  vsTheming:ImageThemingUtilities.ImageBackgroundColor="{Binding Background, RelativeSource={RelativeSource Self}, Converter={StaticResource BrushToColorConverter}}">
+            <DataGrid.Columns>
+                <DataGridTemplateColumn CanUserSort="False" CanUserResize="False" Width="30">
+                    <DataGridTemplateColumn.HeaderStyle>
+                        <Style TargetType="DataGridColumnHeader" BasedOn="{StaticResource {x:Type DataGridColumnHeader}}">
+                            <Setter Property="ToolTipService.ToolTip" Value="Location navigability status" />
+                        </Style>
+                    </DataGridTemplateColumn.HeaderStyle>
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <Grid>
+                                <vsImaging:CrispImage Height="16" Width="16"
+                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
+                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
+                                <vsImaging:CrispImage Height="16" Width="16"
+                                                      Margin="-5,0,0,0"
+                                                      ToolTipService.ToolTip="Location is navigable. Use double-click or Enter to open the document."
+                                                      Moniker="{x:Static vsCatalog:KnownMonikers.DocumentSource}" 
+                                                      Visibility="{Binding TaintIssueViz.Span, Converter={StaticResource SpanToVisibilityConverter}, Mode=OneWay}"/>
+                            </Grid>
+                        </DataTemplate>
+                    </DataGridTemplateColumn.CellTemplate>
+                </DataGridTemplateColumn>
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.RuleKey" Header="Code" Width="130" MinWidth="54">
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding TaintIssueViz.Issue.RuleKey}" 
+                                       Style="{StaticResource CellTextBlockStyle}" />
+                        </DataTemplate>
+                    </DataGridTemplateColumn.CellTemplate>
+                </DataGridTemplateColumn>
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.Severity" Header="Severity" Width="68"  MinWidth="68">
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding TaintIssueViz.Issue.Severity}" 
```

Comment:
I'm displaying the Sonar Severity directly for now. We can add other columns later if necessary. There's not much point displaying the Type since it will always be "Vulnerability". I think there is a security sub-category we could display, but it would require more work to fetch and process it.

The hotspot list has "Category" and "Priority", but neither has a direct equivalent in "normal" issues.